vms/platformvm/blocks/builder/helpers_test.go

```diff
@@ -250,11 +250,6 @@ func defaultState(
 	if err := state.Commit(); err != nil {
 		panic(err)
 	}
-	state.SetHeight( /*height*/ 0)
-	if err := state.Commit(); err != nil {
-		panic(err)
-	}
-
```

Comment:
Was this just a mistake that we had the same code to `SetHeight` and `Commit` twice?

Reply:
yep. No real consequences on the correctness of the test. I believe I make a faulty rebase/merge during Banff develpment

Reply:
Moved to https://github.com/ava-labs/avalanchego/pull/1554